Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/BackLighting.azsli

```diff
@@ -49,26 +53,45 @@ float3 GetBackLighting(Surface surface, LightingData lightingData, float3 lightI
+                thickness = max(transmissionDistance, surface.transmission.thickness);
                 float transmittance = pow( saturate( dot( lightingData.dirToCamera, -normalize( dirToLight + surface.normal * transmissionParams.z ) ) ), transmissionParams.y ) * transmissionParams.w;
                 float lamberAttenuation = exp(-thickness * transmissionParams.x) * saturate(1.0 - thickness);
                 result = transmittance * lamberAttenuation * lightIntensity;
             }
             break;
 
         case TransmissionMode::ThinObject:
-            // Thin object mode, using thin-film assumption proposed by Jimenez J. et al, 2010, "Real-Time Realistic Skin Translucency"
+            // Thin object mode, based on Jimenez J. et al, 2010, "Real-Time Realistic Skin Translucency"
             // http://www.iryoku.com/translucency/downloads/Real-Time-Realistic-Skin-Translucency.pdf
-
-            float litRatio = 1.0 - shadowRatio;
-            if (litRatio)
+            
             {
-                const float thickness = surface.transmission.thickness * transmissionParams.w;
-                const float3 invScattering = rcp(transmissionParams.xyz);
-                const float falloff = ThinObjectFalloff(surface.normal, dirToLight);
-                result = TransmissionKernel(thickness, invScattering) * falloff * lightIntensity * litRatio;
-            }
+                // transmissionDistance < 0.0f means shadows are not enabled --> avoid unnecessary computation
+                if (transmissionDistance < 0.0f)
+                {
+                    break;
+                }
 
+                // Irradiance arround surface point. 
+                // Increase angle of influence (angle(N,L) -> angle(N,L) + acos(transmissionNdLBias)) to smooth transition regions
+                float3 E = surface.albedo * saturate(lightingData.transmissionNdLBias + dot(-surface.normal, dirToLight));
+
+                // Transmission distance modulated by hardcoded constant C and the thickness exposed parameter (in this case modulating the distance traversed by the light inside the object)
+                const float C = 300.0f;
+                float s = transmissionDistance * C * surface.transmission.thickness;
+                
+                // Use scattering color to weight thin object transmission color
+                const float3 invScattering = rcp(max(surface.transmission.scatterDistance, float(0.00001)));
+                
+                // Albedo at front (surface point) is used to approximate irradiance at the back of the object
+                // See observation 4 in [Jimenez J. et al, 2010] 
+                result = TransmissionKernel(s, invScattering) * lightIntensity * E * transmissionParams.w;
+                
+                // Alternative specific to skin translucency
+                // result = T(s) * lightIntensity * E * transmissionParams.w;
```

Comment:
Could you enable that by default through a preprocessor define for the skin material?

Reply:
Sure, makes sense! Done in https://github.com/o3de/o3de/pull/6428/commits/9c60bd008cbda990a237a54eaf71244cca141638.